html/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Tic Tac Toe</title>
+	<link rel="stylesheet" type="text/css" href="../css/style.css">
+</head>
+<body>
+<header>Tic Tac Toe</header>
+<center>
```

Comment:
This tag works on some browsers, but it's deprecated in HTML5. We'll be going over Flexbox this week, which is a more straightforward way to center/arrange items.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center